Chutzpah/TestRunner.cs

```diff
@@ -296,7 +296,19 @@ private bool PerformBatchCompile(ITestMethodRunnerCallback callback, IEnumerable
                                 "Launching test harness '{0}' for file '{1}' in a browser",
                                 testContext.TestHarnessPath,
                                 testContext.FirstInputTestFile);
-                            process.LaunchFileInBrowser(testContext.TestHarnessPath, options.BrowserName);
+
+                            // Allow override from command line.
+                            var browserArgs = testContext.TestFileSettings.BrowserArguments;
+                            if (!string.IsNullOrWhiteSpace(options.BrowserArgs))
+                            {
+                                var path = BrowserPathHelper.GetBrowserPath(options.BrowserName);

```

Comment:
This line should be outside of the if check since a user should be allowed to specify browser name but not browserargs